recipes/quickfast/all/conanfile.py

```diff
@@ -0,0 +1,208 @@
+    exports_sources = "patches/**"
+    _msbuild = None
+    _env_build = None
+    _args = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    @property
+    def _visual_studio_platform(self):
+        if self.settings.arch == "x86":
+            return "Win32"
+        elif self.settings.arch == "x86_64":
+            return "x64"
+        else:
+            raise ConanInvalidConfiguration("QuickFAST cannot be built on Visual Studio for the {} architecture"
+                                            .format(self.settings.arch))
+
+    @property
+    def _wss_platform(self):
+        subsystem = self.settings.get_safe("os.subsystem")
+        if subsystem:
+            raise ConanInvalidConfiguration("QuickFAST cannot be built for Windows subsytems ({})"
+                                            .format(subsystem))
+        return "mingw_msys"
+
+    @property
+    def _platform(self):
+        if self.settings.compiler == "Visual Studio":
+            return self._visual_studio_platform
+        elif self.settings.os == 'Macos':
+            return 'macosx'
+        elif self.settings.os == 'Linux':
+            return 'linux'
+        elif self.settings.os == 'Windows':
+            return self._wss_platform
+        else:
+            raise ConanInvalidConfiguration("QuickFAST cannot be built for the {} operating system"
+                                            .format(self.settings.os))
+
+    @property
+    def _visual_studio_build_type(self):
+        compiler = self.settings.compiler
+        version = Version(compiler.version.value)
+
+        if "8" <= version <= "14":
+            return "v" + version
+        elif version == "14":
+            return "vs2017"
+        elif version == "16":
+            return "vs2019"
+        else:
+            raise ConanInvalidConfiguration("QuickFAST cannot be built with Visual Studio version {}"
+                                            .format(version))
+
+    @property
+    def _build_type(self):
+        if self.settings.compiler == "Visual Studio":
+            return self._visual_studio_build_type
+        else:
+            return 'make'
+
+    @property
+    def _core_value_template_linkflags(self):
+        if self.settings.os == "Macos":
+            return ' -value_template "linkflags+=-framework CoreServices -framework CoreFoundation"'
+        return ''
+
+    @property
+    def _mwc_command_line(self):
+        link_flag = ' -static' if not self.options.shared else ' '
+        fpic_flag = '-fPIC' if self.options.get_safe("fPIC", default=True) else ''
+        platform = self._platform
+        build_type = self._build_type
+        core_value_template_linkflags = self._core_value_template_linkflags
+
+        return 'perl -S mwc.pl -type {} -exclude src/Examples'.format(build_type) + \
```

Comment:
perl for Windows requires strawberryperl/5.30.0.1

Reply:
I added strawberryperl as requirement for mwc (makefile-project-workspace-creator). Should I added it here also?